File: backend-dotnet/Application/Commands/Posts/UpdatePostCommand.cs
using Application.DTOs;
using Domain.Entities;

namespace Application.Commands.Posts
{
    public class UpdatePostCommand : ICommand<Post>
    {
        public Guid PostId { get; set; }
        public UpdatePostDto Dto { get; set; } = null!;
        public Guid UserId { get; set; }
    }
}

File: backend-java/src/main/java/com/jerrygram/application/commands/posts/UpdatePostCommand.java
package com.jerrygram.application.commands.posts;

import com.jerrygram.application.dtos.UpdatePostDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UpdatePostCommand {
    private UUID postId;
    private UUID userId;
    private UpdatePostDto updatePostDto;
}